infra/data-pipeline.yml

```diff
@@ -1275,6 +1275,44 @@ tasks:
         owner: release-mgmt-analysis@mozilla.com
         source: ${repository}/raw/master/data-pipeline.yml
 
+    - ID: train-worksforme
+      created: { $fromNow: "" }
+      deadline: { $fromNow: "3 days" }
+      expires: { $fromNow: "1 year" }
+      provisionerId: proj-bugbug
+      workerType: compute-smaller
```

Comment:
We should try to determine if `compute-smaller` is enough to train the model.

Reply:
compute-small works